# HG changeset patch
# User Markus Mützel <markus.muetzel@gmx.de>
# Date 1552079192 -3600
#      Fri Mar 08 22:06:32 2019 +0100
# Node ID dfd0a2cdb6212de7ef4b04f1083bb112d1c0fe54
# Parent  041caa61ed340039c5e6587a53b1b670b444c27f
Fix fopen test failing on Windows.

* io.tst: Don't use line break which is platform dependent.

diff --git a/test/io.tst b/test/io.tst
--- a/test/io.tst
+++ b/test/io.tst
@@ -654,27 +654,27 @@
 %!test   # write to and read from file with encoding
 %! temp_file = [tempname(), ".txt"];
 %! fid = fopen (temp_file, "wt", "n", "iso-8859-1");
 %! unwind_protect
 %!   [name, mode, arch, codepage] = fopen (fid);
 %!   assert (name, temp_file);
 %!   assert (mode, "w");
 %!   assert (codepage, "iso-8859-1");
-%!   fprintf (fid, "aäu %s\n", "AÄU");
+%!   fprintf (fid, "aäu %s", "AÄU");
 %!   fclose (fid);
 %!   # open in binary mode
 %!   fid2 = fopen (temp_file, "rb");
 %!   [name, mode, arch, codepage] = fopen (fid2);
 %!   assert (name, temp_file);
 %!   assert (mode, "rb");
 %!   assert (codepage, "utf-8");
 %!   read_binary = fread (fid2);
 %!   fclose (fid2);
-%!   assert (read_binary, [97 228 117 32 65 196 85 10].');
+%!   assert (read_binary, [97 228 117 32 65 196 85].');
 %!   # open in text mode with correct encoding
 %!   fid3 = fopen (temp_file, "rt", "n", "iso-8859-1");
 %!   read_text = fscanf (fid3, "%s");
 %!   fclose (fid3);
 %!   assert (read_text, "aäuAÄU");
 %! unwind_protect_cleanup
 %!   unlink (temp_file);
 %! end_unwind_protect
